.gitignore

```diff
@@ -1 +1,3 @@
-.idea/
\ No newline at end of file
+.idea/
+web/coverage/
+node_modules/
```

Comment:
It looks like you added node_modules to the .gitignore after the branch was pushed and it didn't remove them from the repo? I had the same problem (but only with 1 file). I spent about an hour googling how to remove it and nothing seemed to work, but then it was magically gone. Sad to say I'm not sure which command did it, but there is a lot of talk about the best way to do this on the interwebs. That said, I think it shouldn't be a problem 🤷‍♀️

Reply:
Yes, Im happy to chat about it during our standup to see if this is something I should try to avoid. Did the same, tried creating new branches and adding .gitingore retroactively to no avail.